code/unclassified/greatest_digit

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+using namespace std;
+
+int greatest_digit(int n){
+  int max=0;
+  while(n>=10){
```

Comment:
Infinite loop as `n` is not being modified. Kindly correct to get this merged 👍

Reply:
thanks